package.json

```diff
@@ -8,8 +8,20 @@
     "postinstall": "lerna bootstrap --ci --force-local",
     "postpublish": "lerna exec -- npm install --package-lock-only && git commit -a -m 'Bump package-lock' && git push"
   },
-  "devDependencies": {},
   "dependencies": {
     "lerna": "^3.22.1"
+  },
+  "devDependencies": {
+    "@contrast/eslint-config": "^2.0.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-mocha": "^6.3.0",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "prettier": "^1.19.1"
+  },
+  "prettier": {
```

Comment:
aren't these already in the `@contrast/eslint-config`?

Reply:
They are, but there can be race conditions between prettier and eslint when both are set to autofix that cause them never to agree. This isn't a problem when running the eslint-prettier integration, but it can be with editor integrations.